balboa-client/src/main/scala/com/socrata/balboa/impl/HashMapBuffer.scala

```diff
@@ -8,8 +8,8 @@ import com.socrata.metrics.MetricQueue
 import com.socrata.metrics.components.{BufferComponent, BufferItem, MessageQueueComponent}
 import org.slf4j.LoggerFactory
 
+import collection.mutable
```

Comment:
Don't have to for Scala packages. I didn't do it on purpose, Intellij auto imported. I kind of like saying 'scala', so I'll fix that.